keep-ui/app/navbar.tsx

```diff
@@ -6,10 +6,17 @@ import { useSession } from "next-auth/react";
 import { CMDK } from "./command-menu";
 
 export default function Navbar() {
-  const { data: session } = useSession();
+  // extract status and session properties from useSession hook
+  const { status, data: session } = useSession();
+  // if session is loading, return nothing
+  if (status === "loading") return <></>;
+  if (status === "unauthenticated") return <></>;
+  if (!session) return <></>;
```

Comment:
is that required? assuming you're checking everything inside NavbarInner

Reply:
yea since otherwise the navbar will try to render before having session